modules/snooker/src/test/ts/browser/CopyAttributesTest.ts

```diff
@@ -0,0 +1,40 @@
+import { Log, Pipeline, Step } from '@ephox/agar';
+import { Assert, UnitTest } from '@ephox/bedrock-client';
+import { Fun, Optional } from '@ephox/katamari';
+import { SelectorFind, SugarDocument, SugarElement } from '@ephox/sugar';
+
+import { cellOperations } from 'ephox/snooker/api/TableFill';
+
+UnitTest.asynctest('modules.snooker.src.test.ts.browser.CopyAttributesTest.ts', (success, failure) => {
+  const operations = cellOperations(Fun.noop, SugarDocument.getDocument(), Optional.none());
+
+  const generateCell = (input: string) => {
+    const modifiedInput = '<table><tbody><tr>' + input + '</tr></tbody></table>';
+    const element = SugarElement.fromHtml<HTMLTableElement>(modifiedInput);
+    return SelectorFind.descendant<HTMLTableCellElement>(element, 'td').getOrDie('Error here');
+  };
+
+  const sMakeCellComparison = (id: string, desctripion: string, input: string, expectedOutput: string): Step<any, any> =>
+    Log.step(id, desctripion, Step.sync(() => {
+      const newCell = operations.cell({
+        element: generateCell(input),
+        colspan: 0,
+        rowspan: 0
+      });
+
+      Assert.eq(
+        '',
+        expectedOutput,
+        newCell.dom.outerHTML
+      );
+    }));
+
+  Pipeline.async({}, [
+    sMakeCellComparison('TINY-6485', 'nothing copied if nothing is added', '<td></td>', '<td><br></td>'),
```

Comment:
There are no tests here for `th` elements, which are primarily what we want to test the `scope` attribute is cloned for, since while `scope` is allowed on a `td`, it's deprecated. So could you please ensure most are `th` instead of `td` (maybe just have one or two td elements?)